pyneuromodulation/tests/test.py

```diff
@@ -79,9 +79,9 @@ def initialize_rereference(df_M1):
 
     to_ref_idx = np.array(df_M1[(df_M1['used'] == 1)].index)
 
-    cortex_idx = np.where(df_M1.ECOG == 1)[0]
-    subcortex_idx = np.array(df_M1[(df_M1["ECOG"] == 0) & \
-                    (df_M1['used'] == 1) & (df_M1['target'] == 0)].index)
+    cortex_idx = np.where(df_M1.type == 'ecog')[0]
+    subcortex_idx = np.array(df_M1[(df_M1["type"] == 'seeg') | (df_M1['type'] == 'dbs')
+                                   & (df_M1['type'] == 'lfp')].index)
```

Comment:
@timonmerk  Same as above:
Shouldn't there be a logical "or" before the "lfp" typechecke like: 
`| (df_M1['type'] == 'lfp')`